.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor/pkg
-runc
+/runc
+Godeps/_workspace/src/github.com/opencontainers/runc
```

Comment:
If you'd like to ignore runc in both locations, I guess you could just leave .gitignore untouched, no?